### DISCUSSION TITLE
prettier plugin for prisma schema?

### DISCUSSION THREAD

**Discussion by liesislukas** [prisma/prisma#14718] [Q&A]
would love to see some plugin for prettier to keep spacing formatting consistent inside prisma schema. Any thought about it?

https://prettier.io/docs/en/plugins.html#community-plugins

Something like this:

![image](https://user-images.githubusercontent.com/2733862/183621948-e2437e18-5fab-46a8-998d-0bf744889d5f.png)

would be aligned properly by IDE on save or with CI/CD pipe

![image](https://user-images.githubusercontent.com/2733862/183622087-637298ea-0289-413b-8f87-37307b940a21.png)

**Comment by annibuliful** [comment]
Does this one help you?
https://marketplace.visualstudio.com/items?itemName=Prisma.prisma

This is my setting for prisma
```
  "[prisma]": {
    "editor.defaultFormatter": "Prisma.prisma"
  },
```

**Comment by janpio** [comment]
Does this work?
https://www.npmjs.com/package/prettier-plugin-prisma

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
